nemo_rl/algorithms/grpo.py

```diff
@@ -604,13 +605,33 @@ def grpo_train(
                     )
 
             with timer.time("data_processing"):
+                use_overlong_filtering = master_config["grpo"].get(
```

Comment:
We've banned using default dict args inside of the codebase (we should probably write this fact down somewhere @terrykong). Can you switch this to just call the arg directly? If a config is missing this key, it should crash.